### PR TITLE
Reduce logging

### PR DESCRIPTION
**Pull request by dbemiller**
If there are good standards for when to log what, I don't know them... so I just made some guesses. Lemme know if I'm off the mark.

### CODE REVIEW

File: pbs/usersync.go [hunk 1]
```diff
@@ -157,7 +157,9 @@ func OptOut(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 
 	err := VerifyRecaptcha(rr)
 	if err != nil {
-		glog.Infof("Optout failed recaptcha: %v", err)
+		if glog.V(2) {
```
Comment (mkendall07) [inline pbs/usersync.go]: I think we should just use the standard log levels and change the `minloglevel ` flag to 1 (warn). I don't see the need for custom levels. This assumes we have the appropriate levels used in the app, which shouldn't be hard to fix.

File: pbs_light.go [hunk 2]
```diff
@@ -220,7 +222,9 @@ func auction(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 
 	pbs_req, err := pbs.ParsePBSRequest(r, dataCache)
 	if err != nil {
-		glog.Info("error parsing request", err)
+		if glog.V(2) {
+			glog.Infof("Failed to parse /auction request: %v", err)
```
Comment (mkendall07) [inline pbs_light.go]: Probably should be a warn level.

File: pbs_light.go [hunk 3]
```diff
@@ -424,40 +430,8 @@ func auction(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
 		}
 	}
 
-	if glog.V(1) {
-		glog.Infof("Request for %d ad units on url %s by account %s got %d bids", len(pbs_req.AdUnits), pbs_req.Url, pbs_req.AccountID, len(pbs_resp.Bids))
```
Comment (mkendall07) [inline pbs_light.go]: I'd leave this here at the info level.

File: pbs/usersync.go (hunk 1 above)
Comment (dbemiller) [inline pbs/usersync.go]: Irritating as it is... I think `minloglevel` is only supported by the C++ version of `glog`. The Go docs [don't mention it](https://godoc.org/github.com/golang/glog), and the server fails with an error message if I try to use it.

I don't see any way to filter out `Info` messages in Go's version of glog. Only based on `V(#)` levels

File: pbs_light.go (hunk 3 above)
Comment (dbemiller) [inline pbs_light.go]: This I can do. I'll bump it up to `V(2)`

File: pbs/usersync.go (hunk 1 above)
Comment (mkendall07) [reply pbs/usersync.go]: such a bummer.

**Pull request merged**